src/pages/about-us.js

```diff
@@ -3,12 +3,21 @@ import { StaticQuery, graphql } from 'gatsby'
 
 import Layout from '../components/layout'
 import Head from '../components/Common/Head'
+import SubsidiariesSection from '../components/AboutUs/SubsidiariesSection'
 
-const AboutUs = ({ data: { contentfulAboutUsPage: content } }) => (
-  <Layout>
-    <Head page={content} />
-  </Layout>
-)
+const AboutUs = ({ data: { contentfulAboutUsPage: content } }) => {
+  const { yldGroupTitle, subsidiaries } = content
+
+  return (
+    <Layout>
+      <Head page={content} />
+      <SubsidiariesSection
+        yldGroupTitle={yldGroupTitle}
```

Comment:
`yldGroupTitle` could be replaced by a simple `title` prob since it'll be used in the Subsidiaries component scope, so there's no need to create more complex prop names, what do you think?

Reply:
Haha same as last PR. I'll learn :wink: 

Thanks!